src/zabapgit_forms.prog.abap

```diff
@@ -137,6 +137,14 @@ FORM package_popup TABLES   tt_fields TYPE ty_sval_tt
   ENDIF.
 ENDFORM.                    "package_popup
 
+FORM branch_to_transport_popup TABLES   tt_fields TYPE ty_sval_tt
+                        USING    pv_code TYPE clike
+                        CHANGING cs_error TYPE svale
+                                 cv_show_popup TYPE c
+                        RAISING  lcx_exception ##called ##needed.
+* called dynamically from function module POPUP_GET_VALUES_USER_BUTTONS
+ENDFORM.
+
```

Comment:
but there are no buttons in the popup? perhaps change it to a different function module?